src/graphql/root/query/btc-price.ts

```diff
@@ -19,7 +30,7 @@ const BtcPriceQuery = GT.Field({
       formattedAmount: price.toString(),
       base: Math.round(price * 10 ** SAT_PRICE_PRECISION_OFFSET),
       offset: SAT_PRICE_PRECISION_OFFSET,
-      currencyUnit: "USDCENT",
+      currencyUnit: `${currency}CENT`,
```

Comment:
Should we use major/minor unit here or something similar? Some currencies don't have "cents" I think